module/VuFind/src/VuFindTest/Integration/MinkTestCase.php

```diff
@@ -231,7 +231,7 @@ protected function changeYamlConfigs($configs, $replace = [])
      *
      * @return void
      */
-    protected function changeConfigFile($configName, $settings, $replace = false)
+    protected function changeConfigFile(string $configName, array $settings, bool $replace = false)
```

Comment:
If we're adding types, might as well do the return value too.

```suggestion
    protected function changeConfigFile(string $configName, array $settings, bool $replace = false): void
```